src/api/lib/activexml/node.rb

```diff
@@ -480,10 +480,10 @@ def find_matching(conds)
       return
```

Comment:
You have a typo in your commit message.

The commit header should be something like "Make method private". Right now it's sounds like you actually change the method('s behaviour).

Also what is "_stay away from this_" supposed to mean?:thinking:

Reply:
@bgeuken 

> Also what is "stay away from this" supposed to mean?

that it is private hahaha